src/main/kotlin/net/atos/zac/app/zaak/ZaakRestService.kt

```diff
@@ -281,14 +281,16 @@ class ZaakRestService @Inject constructor(
             val user = identityService.readUser(it.id)
             zrcClientService.updateRol(zaak, zaakService.bepaalRolMedewerker(user, zaak), AANMAKEN_ZAAK_REDEN)
         }
-        cmmnService.startCase(
-            zaak,
-            zaaktype,
-            zaakafhandelParameterService.readZaakafhandelParameters(
-                URIUtil.parseUUIDFromResourceURI(zaaktype.url)
-            ),
-            null
-        )
+        if (!bpmnService.startProcess(zaak, zaaktype, null)) {
```

Comment:
The link between the zaaktype and the BPMN process flow is in the usage of the zaaktype property referentieproces.naam. This may not be appropriate but we can decide on that later.
It is not possible to start a CMMN case other then the existing generic one.
I have refactored to code in order to clarify he decision to start a BPMN proces or a CMMN case